src/conf/startup.properties

```diff
@@ -136,6 +136,12 @@ prism.configstore.listeners=org.apache.falcon.entity.v0.EntityGraph,\
 
 *.falcon.cleanup.service.frequency=days(1)
 
+# Default number of threads to be used to restore entities.
+*.falcon.default.threads.load.restore.entities=100
+
+# Default timeout in minutes to load entities
+*.falcon.default.timeout.minutes=30
```

Comment:
Is assigning the 30 minutes timeout reasonable. I am of the opinion it must be less.